node.js webpack experimental support (#37458)
DataDog/documentation: content/en/tracing/trace_collection/dd_libraries/nodejs.md

})
```

> **Note**: Every application is different and you may need to experiment with the list of `external` packages.

Reviewer: I'm not sure how helpful this note is; is there more guidance we can provide here? What does "experiment" mean in this case?

Author: I removed the note lines since it's stuff that the user is likely to figure out on their own.

#### Bundling with Webpack

This library provides experimental Webpack support in the form of a Webpack plugin. To use the plugin, make sure you have `dd-trace@5.94.0+` installed, and then require the `dd-trace/webpack` module when building your bundle.

> Here's an example of how one might use `dd-trace` with Webpack:

Reviewer: ```suggestion
The following example shows how to use `dd-trace` with Webpack:
```
